prowler/providers/azure/azure_provider.py

```diff
@@ -155,14 +215,30 @@ def get_output_mapping(self):
 
     # TODO: this should be moved to the argparse, if not we need to enforce it from the Provider
     # previously was using the AzureException
+    @staticmethod
     def validate_arguments(
-        self, az_cli_auth, sp_env_auth, browser_auth, managed_entity_auth, tenant_id
+        az_cli_auth, sp_env_auth, browser_auth, managed_identity_auth, tenant_id
     ):
+        """
+        Validates the authentication arguments for the Azure provider.
+
+        Args:
+            az_cli_auth (bool): Flag indicating whether AZ CLI authentication is enabled.
+            sp_env_auth (bool): Flag indicating whether Service Principal environment authentication is enabled.
+            browser_auth (bool): Flag indicating whether browser authentication is enabled.
+            managed_identity_auth (bool): Flag indicating whether managed identity authentication is enabled.
+            tenant_id (str): The Azure Tenant ID.
+
+        Raises:
+            SystemExit: If none of the authentication methods are set.
```

Comment:
Not sure about this exception type but since we will work on our custom exceptions, this is fine. I know that this was not introduced in this PR.

Reply:
Ok